### PR TITLE
leporazine medi-pens mining

### PR DESCRIPTION
**Pull request by LanCartwright**
because someone buffed basilisks and they're cancer to fight

:cl:
add: Adds Leporazine Medi-Pens to Mining Vendor for 50 points.
add: Adds Value pack of 5 Leporazine Medi-Pens in a box to the Mining Vendor for 200 points.
add: Leporazine Medi-Pens have 10u of Leporazine and 10u of Oxandrolone each.
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
Thanks

**Comment by LanCartwright** [comment]
who changed basilisks to freeze beam per hit anyway god damn

**Comment by Zilenan91** [comment]
They've always done that afaik

**Comment by LanCartwright** [comment]
nope, not on an older tg branch, I've tested.

At melee range they only used melee attacks.

**Comment by Jordie0608** [comment]
Needs a changelog

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -104,6 +104,14 @@
 	amount_per_transfer_from_this = 20
 	list_reagents = list("ephedrine" = 10, "coffee" = 10)
 
+/obj/item/weapon/reagent_containers/hypospray/medipen/leppen //basilisks were buffed
+	name = "leporazine medipen"
+	desc = "A rapid way to stabalise your body's temperature and reducing burn wounds."
```
Comment (Xhuis) [inline code/modules/reagents/reagent_containers/hypospray.dm]: reduce*

Comment (bgobandit) [inline code/modules/reagents/reagent_containers/hypospray.dm]: *stabilize

**Comment by LanCartwright** [comment]
Adds Leporazine Medi-Pens to Mining Vendor for 50 points.
Adds Value pack of 5 Leporazine Medi-Pens in a box to the Mining Vendor for 200 points.

Leporazine Medi-Pens have 10u of Leporazine and 10u of Oxandrolone each.

File: code/modules/mining/equipment_locker.dm [hunk 2]
```diff
@@ -278,7 +278,9 @@
 	var/list/prize_list = list(
 		new /datum/data/mining_equipment("Stimpack",			/obj/item/weapon/reagent_containers/hypospray/medipen/stimpack,	    50),
 		new /datum/data/mining_equipment("Stimpack Bundle",		/obj/item/weapon/storage/box/medipens/utility,	 				   200),
-		new /datum/data/mining_equipment("Whiskey",             /obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey,    100),
+		new /datum/data/mining_equipment("Leporazine Medipen",	/obj/item/weapon/reagent_containers/hypospray/medipen/leppen,		50),
+		new /datum/data/mining_equipment("Leppen Bundle",		/obj/item/weapon/storage/box/medipens/leppen,						200),
+		new /datum/data/mining_equipment("Whiskey",				/obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey,    100),
```
Comment (tkdrg) [inline code/modules/mining/equipment_locker.dm]: The whiskey should be above the bundle here

**Comment by tkdrg** [comment]
He meant putting the changelog on the OP. I'll edit this one for you so you see how it works

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -104,6 +104,14 @@
 	amount_per_transfer_from_this = 20
 	list_reagents = list("ephedrine" = 10, "coffee" = 10)
 
+/obj/item/weapon/reagent_containers/hypospray/medipen/leppen //basilisks were buffed
```
Comment (tkdrg) [inline code/modules/reagents/reagent_containers/hypospray.dm]: This comment is pretty needless

Comment (KazeEspada) [inline code/modules/reagents/reagent_containers/hypospray.dm]: So are half the other ones, but got make sure its written meme like "i ded pls nerf basilisks"

Comment (LanCartwright) [inline code/modules/reagents/reagent_containers/hypospray.dm]: I actually haven't died to Basilisks ever, but the recent buff makes it nearly impossible to fight en-masse.

Before the buff, you could easily kill basilisks, but the buff changed it so it's nearly impossible due to the incredibly delayed temperature burn damage.

**Comment by tkdrg** [comment]
Now conflicts too

**Comment by tkdrg** [comment]
ping me if you decide to fix this

**Pull request closed without merging**